app/javascript/scavenger_hunt/scenes/Questions/QuestionSeven/QuestionSeven.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { TextField, Grid } from '@material-ui/core';
+import { Card, Button } from "react-bootstrap";
+import './QuestionSeven.scss';
+
+import Map from '../_images/map.png';
+
+function QuestionSeven () {
```

Comment:
we try to do `const QuestionSeven = () =>{}` instead of using `function`

i think it looks cleaner and is more consistent

Reply:
I agree with Ryan

Reply:
fixed